apps/web/server/lib/ssg.ts

```diff
@@ -36,8 +36,5 @@ export async function ssgInit<TParams extends { locale?: string }>(opts: GetStat
     },
   });
 
-  // always preload i18n
-  await ssg.viewer.public.i18n.fetch();
```

Comment:
We shouldn't prefetch i18n in SSG because we don't know the user locale and we endup prefetching wrong locale data.

It becomes a problem because useViewerI18n is configured to have a staleTime infinite for this query and thus it reuses the english language locale data and never fetches again. So, if someone reloads on an SSG page currently, the app language remains english throughout the app soft navigations